systems/plants/RigidBodyManipulator.cpp

```diff
@@ -674,7 +674,10 @@ void RigidBodyManipulator::doKinematics(MatrixBase<DerivedA>  & q, bool b_comput
 void RigidBodyManipulator::doKinematics(double* q, bool b_compute_second_derivatives, double* qd)
```

Comment:
i think we should simply zap this old interface.  is anything depending on it still?